Gdk-3.0 supports multiple targets, so the var changed from "target" to "targets".
However, it now lists *all* the targets Gdk is built-in with, so this patch is
COMPLETELY wrong. Someone needs to do an actual fix & file an upstream bug for it

---
--- a/configure.in
+++ b/configure.in
@@ -163,7 +163,7 @@
 done
 
 dnl Check the smclient backend
-GDK_TARGET="$($PKG_CONFIG --variable target gdk-3.0)"
+GDK_TARGET="$($PKG_CONFIG --variable targets gdk-3.0)"
 
 SMCLIENT_PKGS=
 AC_MSG_CHECKING([which smclient backend to use])
@@ -215,7 +215,7 @@
 dnl X11 related functionality
 dnl *************************
 have_x11=no
-if test x$(pkg-config --variable=target gtk+-3.0) = xx11; then
+if test x$(pkg-config --variable=targets gtk+-3.0) = xx11; then
 	AC_PATH_X
 
 	have_x11=yes
